### DISCUSSION TITLE
ADRG/CADRG data usage question

### DISCUSSION THREAD

**Discussion by miles-tbe** [pelicanmapping/osgearth#2100] [Support Q&A]
How do I specify the use of ADRG/CADRG data in an osgEarth .earth file?

**Comment by gwaldron** [comment]
Have you tried loading it with the `GDALImage` layer?

**Comment by keince** [comment]
The best way to do this is to gather all the CADRG/RPF frames up and then put them into a vrt file using `gdalbuildvrt.exe`. You then convert the resulting vrt file into a GeoTIFF file using `gdal_translate.exe`. The resulting GeoTIFF file is then loaded into a `GDALImage` layer using the usual syntax in the earth file.

If you go this way, I would highly recommend that you add overviews to your GeoTIFF file using `gdaladdo.exe`. That will speed up the loading of the data when osgEarth starts up.

Loading the CADRG/RPF frames directly into the `GDALImage` layer can be done but it results in slow performance, especially for large map coverage.
